modules/getting-started/proc-customize-rhdh-homepage.adoc

```diff
@@ -22,48 +34,73 @@ proxy:
+. Enter the URL of your Git repository into the *Git Repo URL* field.
 +
 --
-To use the `red-hat-developer-hub-customization-provider` service, you can add the URL of https://github.com/redhat-developer/red-hat-developer-hub-customization-provider[red-hat-developer-hub-customization-provider] repository.
+To use the `red-hat-developer-hub-customization-provider` service, add the URL for the https://github.com/redhat-developer/red-hat-developer-hub-customization-provider[red-hat-developer-hub-customization-provider] repository or your fork of the repository containing your customizations.
 --
 
-. In the *General* section, rename the value in the *Name* field to *rhdh-customization-provider* and click *Create*.
-. Go to the *Advanced Options* and copy the value from the *Target Port*.
+. On the *General* tab, enter *red-hat-developer-hub-customization-provider* in the *Name* field and click *Create*.
+. On the *Advanced Options* tab, copy the value from the *Target Port*.
 +
---
-The *Target Port* is used to automatically generate a Kubernetes or OpenShift service to communicate with.
---
+[NOTE]
+====
+The *Target Port* automatically generates a Kubernetes or {ocp-short} service to communicate with.
+====
++
+. Add the following code to the `app-config-rhdh.yaml` file:
++
+[source,yaml]
+----
+proxy:
+  endpoints:
+    # Other Proxies
+    # customize developer hub instance
+    '/developer-hub':
+      target: ${HOMEPAGE_DATA_URL}
+      changeOrigin: true
+      # Change to "false" in case of using self-hosted cluster with a self-signed certificate
+      secure: true
+----
+where `HOMEPAGE_DATA_URL` is defined as `pass:c[http://<SERVICE_NAME>:8080]`, for example, `pass:c[http://rhdh-customization-provider:8080]`.
++
+[NOTE]
+====
+The `red-hat-developer-hub-customization-provider` service contains the 8080 port by default. If you are using a custom port, you can specify it with the 'PORT' environmental variable in the `app-config-rhdh.yaml` file.
+====
++
+. Replace the `HOMEPAGE_DATA_URL` by adding the URL to `rhdh-secrets` or by directly replacing it in your custom ConfigMap.
++
+. Delete the {product-short} pod to ensure that the new configurations are loaded correctly.
 
-. To view the service, navigate to the *OpenShift Administrator* view and go to the *Networking -> Service* section.
+.Verification
+* To view the service, navigate to the *Administrator* perspective in the {ocp-short} web console and click *Networking* > *Service*.
 +
---
+[NOTE]
+====
 You can also view the *Service Resources* in the Topology view.
+====
 
-If you follow this procedure with examples, then `rhdh-customization-provider` service is called and contains the 8080 port. The provided API URL for the Home page must return the data in JSON format as shown in the following example:
-
+* Ensure that the provided API URL for the Home page returns the data in JSON format as shown in the following example:
```

Comment:
should we add similar notes and JSON sample data for techradar and learning paths as well, looks like I missed it before

cc @linfraze @Zaperex

Reply:
I think that would make sense since the expected JSON format is not the same for each. Main concern would be the size of the snippets to fully showcase the format (ex: how to configure multiple, what types of fields are supported, etc.)

Reply:
Yeah, I think that would be helpful to users. Let's add this under a separate Jira/PR.

Reply:
https://issues.redhat.com/browse/RHIDP-3303